odoo/addons/base/models/res_bank.py

```diff
@@ -87,7 +87,7 @@ def _get_supported_account_types(self):
     bank_bic = fields.Char(related='bank_id.bic', readonly=False)
     sequence = fields.Integer(default=10)
     currency_id = fields.Many2one('res.currency', string='Currency')
-    company_id = fields.Many2one('res.company', 'Company', default=lambda self: self.env.company, ondelete='cascade')
+    company_id = fields.Many2one('res.company', 'Company', default=lambda self: self.env.company, ondelete='cascade', readonly=True)
```

Comment:
why adding this readonly? I did not see this in the spec

Reply:
@csnauwaert I did for this point : https://drive.google.com/file/d/1bu0nYE4qd5xVcUMKcLlOXyOpEVed8ADH/view

`# Do not allow creation of bank accounts in other companies than the running one`

Reply:
Ok, did not see this, all seems good then :) just need to be tested one last time